metsrw/utils.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from six.moves.urllib.parse import quote_plus, urlparse, urlunparse
+from six.moves.urllib.parse import quote_plus, unquote_plus, urlparse, urlunparse
```

Comment:
I think this just creeps over the line-length limit.

Reply:
👍